deployment/helm/node-feature-discovery/values.yaml

```
@@ -483,7 +483,7 @@ topologyUpdater:
   annotations: {}
   daemonsetAnnotations: {}
   affinity: {}
-  podSetFingerprint: true
+  disablePodSetFingerprint:
```

I didn't realize pod fingerprint has already been enabled by default in the Helm deployment, ever since the feature was introduced in NFD v0.13.0 🤦‍♂️. The documentation was incorrect. Just keep this as is

```suggestion
  podSetFingerprint: true
```

And generally, we should not break existing users, unless there's a very good reason. Plus, we've made a promise not to break without deprecating first ([deprecation policy](https://kubernetes-sigs.github.io/node-feature-discovery/v0.15/reference/versions.html#configuration-options)).

---

this means we don't need to provide option which disable PodSetFingerprint  in the chart, because it is already enabled in previous chart versions?

---

done

---

done